addons/web/static/src/views/list/list_renderer.js

```diff
@@ -55,6 +55,7 @@ const FIXED_FIELD_COLUMN_WIDTHS = {
     integer: "74px",
     monetary: "104px",
     handle: "33px",
+    many2many_tags: "150px",
```

Comment:
I didn't test it, but I believe this rather stands for width than min-width. For many2many_tags, the width could be larger than 150px if there are several tags, and the table as few columns.